README.md

```diff
@@ -90,9 +90,9 @@ def myCloud = new KubernetesCloudBuilder()
     .withName("mytemplate")
     .withLabel("k8s")
 .endTemplate()
-.build();
+.build()

```

Comment:
Any reasons here we should remove semicolon? I think it is optional in groovy.

Reply:
Exactly. Semicolons are optional in Groovy. Because of this they are not used in common Groovy code. That's why I removed them.